group_vars/location_vaterhaus/networks.yml

```diff
@@ -0,0 +1,85 @@
+---
+ipv6_prefix: "2001:bf7:830:a500::/56"
+
+# Vaterhaus got following prefixes:
+# Router: 10.230.192.192/26
+# --MGMT: 10.230.192.192/27
+# --MESH: 10.230.192.224/27 {as /32}
+# --DHCP: 10.31.155.0/26 
+
+networks:
+  - vid: 10
+    role: mesh
+    name: mesh_zwingli
+    prefix: 10.230.192.224/32
+    ipv6_subprefix: -1
+    ptp: true
+    mesh_metric: 128
+
+  - vid: 11
+    role: mesh
+    name: mesh_w
+    prefix: 10.230.192.225/32
+    ipv6_subprefix: -2
+
+  - vid: 12
+    role: mesh
+    name: mesh_no
+    prefix: 10.230.192.226/32
+    ipv6_subprefix: -3
+    ptp: true
```

Comment:
Is this a point to point or point to multipoint AP?

Reply:
it is ptp - a miktrotik cube